lib/inquisition/outputter/doc/tpl/bottlenecks_detection.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'security/bottlenecks_detection/active_record_doctor'
```

Comment:
It would be better to move all thing related to the `bottlenecks_detection` near the `security` folder